be/src/Project/Web/Buyer/Buyer.Web/Areas/Orders/ApiControllers/BasketCheckoutApiController.cs

```diff
@@ -102,6 +109,7 @@ public async Task<IActionResult> Checkout([FromBody] CheckoutBasketRequestModel
                 deliveryAddress?.CountryId,
                 model.MoreInfo,
                 model.HasCustomOrder,
+                clientApprovlas.Any(x => x.NotificationTypeId == ClientNotificationTypeConstants.ApprovalToSendOrderConfirmationEmailsId),
```

Comment:
**suggestion:** Please move CheckoutBasketAsync  method after deliveryAddresses statement and after that fix problem with nullable billing and delivery address